jiant/proj/main/modeling/primary.py

```diff
@@ -154,6 +164,19 @@ def get_hidden_size(self):
     def get_hidden_dropout_prob(self):
         return self.config.hidden_dropout_prob
 
+    def __call__(self, input_ids, segment_ids, input_mask, output_hidden_states=True):
```

Comment:
I might have missed this in the previous PR - what's the reasons for modifying the `__class__` and `__call__` of the object? I would prefer a more transparent interface (e.g. `.encode`)

Reply:
It was meant to mimic HF Transformer's forward but I am not attached to any specific interface.